libs/SalesforceSDK/src/com/salesforce/androidsdk/config/BootConfig.java

```diff
@@ -63,11 +66,14 @@
 	private static final String SHOULD_AUTHENTICATE = "shouldAuthenticate";
 	private static final String ATTEMPT_OFFLINE_LOAD = "attemptOfflineLoad";
 	private static final String PUSH_NOTIFICATION_CLIENT_ID = "androidPushNotificationClientId";
+	private static final String UNAUTHENTICATED_START_PAGE = "unauthenticatedStartPage";
 
 	// Default for optional configs.
 	private static final boolean DEFAULT_SHOULD_AUTHENTICATE = true;
 	private static final boolean DEFAULT_ATTEMPT_OFFLINE_LOAD = true;
 
+	private boolean configIsHybrid;
```

Comment:
Added this property for unit test support.

Reply:
Do we need this? Can we use `SalesforceSDKManager.getInstance().isHybrid()` instead to make this determination? That flag would automatically be set by the corresponding init call used in that test (`initHybrid()` or `initNative()`).

Reply:
It's for testing.  Without it, what can be an isolated unit test in the com.salesforce.androidsdk package, instead requires loading the tests in the context of a SalesforceHybridSDKManager instance, which in turn requires setting up an app context to load the manager.  We could do that, but it seems like overkill for simple unit test functionality that doesn't otherwise depend on any of that setup.

Reply:
Makes sense.